docs/users/troubleshooting.md

```diff
@@ -202,10 +202,10 @@ For example, if your Dockerfile were
 ```dockerfile
 ARG BASE_IMAGE
 FROM $BASE_IMAGE
-RUN npm install --global gulp-cli
+RUN npm install --global forever
```

Comment:
The second commit seems unrelated to the PR.
Careful when merging.

Reply:
Thanks so much @jonaseberle - I happened to be testing a trivial Dockerfile to figure out what was going on with https://github.com/docker/compose/issues/9277 (and https://github.com/docker/compose/issues/9276) and so was actually looking at the example and docs. It turned out that the Dockerfile.example and docs were outdated because gulp-cli has long been installed by default. So I just fixed the example and docs.